src/buildercore/fastly/vcl/main.vcl

```diff
@@ -29,13 +39,6 @@ sub vcl_fetch {
     return(pass);
   }
 
-  if (beresp.status == 500 || beresp.status == 503) {
-    set req.http.Fastly-Cachetype = "ERROR";
```

Comment:
no idea what this header was, just placed there for logging?

Reply:
Not sure. Is part of the boilerplate.